.changeset/quiet-walls-repair.md

```diff
@@ -0,0 +1,5 @@
+---
+'@api3/airnode-utilities': patch
```

Comment:
One more minor nit: I think this should be minor instead of patch given it is a change in behaviour rather than a bug fix.